### DISCUSSION TITLE
Where to run the management cluster cheaply? Is minikube an option?

### DISCUSSION THREAD

**Discussion by smulikHakipod** [berops/claudie#1343] [General] [answered]
Hi!
What is the best solution to run the claudie managment k8s cluster?
I am not taking about the clusters claudie creates and the nodepool, which can be hosted in various ways - I am talking about the management cluster.

Is minikube an option? If so, how is the backup/persistency plan here?
If not - what are some cheap options? I rather don't pay dozens of dollar at least a month for managed k8s solution in one of the cloud providers.
I thought of creating a Mini PC with k3s - but it really seems like a overkill for the problem.  

What are your thoughts here?

Thanks!

**Comment by smulikHakipod** [comment]
After further reading the docs, it seems the recommended way to use kind. Probably could be installed on cheap 6$ a month VPS as well, which don't require expensive managed clusters. Just need to make sure to backup it as https://github.com/berops/claudie/issues/1349 indicates

**Comment by Despire** [comment, marked as answer]
Hi @smulikHakipod 

You can use kind or minikube to run claudie if you don't want to run the management cluster in a cloud provider.
The backup procedure for kind and minkube or other lightweight kubernetes distributions is described in https://docs.claudie.io/latest/creating-claudie-backup/creating-claudie-backup/.

Note that you will need to use the **manual backup** procedure described, as kind or minikube use hostpath volumes, which velero does not currently backup.
